meilisearch/client.py

```diff
@@ -454,6 +454,7 @@ def create_dump(self) -> TaskInfo:
         task = self.http.post(self.config.paths.dumps)
 
         return TaskInfo(**task)
+    
```

Comment:
There should only be one space between class methods, this is one of the `black` errors. If you run `black meilisearch tests` it will fix this.